framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -505,6 +518,29 @@ private void parseMethod(MethodDeclaration decl, ExecutableElement elt,
         putNew(atypes, elt, methodType);
     }
 
+    private void handleExistingAnnotations(AnnotatedTypeMirror atype, Type typeDef) {
```

Comment:
Could you please add documentation for this method, indicating what its purpose or effect is?